nvidia_gpu_stats/src/gpu_nvidia.rs

```diff
@@ -4,6 +4,12 @@ use nvml_wrapper::error::NvmlError;
 use nvml_wrapper::{Device, Nvml};
 use sysinfo::{Pid, System};
 
+#[cfg(target_os = "windows")]
+const LIB_PATH: &str = "nvml.dll";
```

Comment:
ran into this error when testing in CircleCI's win gpu machines:
Error: LibloadingError(LoadLibraryExW { source: Os { code: 126, kind: Uncategorized, message: "The specified module could not be found." } })
(nvml.dll lives under /c/Program Files/NVIDIA Corporation/NVSMI/nvml.dll)
probably libloading can't find this or something.